apps/designer/app/canvas/features/text-editor/index.ts

```diff
@@ -0,0 +1,3 @@
+export { TextEditor } from "./text-editor";
+// hack for React.lazy
```

Comment:
What the hack?

Reply:
Please add a more specific comment, I have no idea :)